tests/system/Cookie/CookieTest.php

```diff
@@ -173,24 +174,40 @@ public function testSetCookieHeaderCreation(string $header, array $changed): voi
 	public static function setCookieHeaderProvider(): iterable
 	{
 		yield 'basic' => [
-			'test=value',
-			['name' => 'test', 'value' => 'value'],
-		];
+				  'test=value',
+				  [
+					  'name'  => 'test',
+					  'value' => 'value',
+				  ],
+			  ];
 
 		yield 'empty-value' => [
-			'test',
-			['name' => 'test', 'value' => ''],
-		];
+				  'test',
+				  [
+					  'name'  => 'test',
+					  'value' => '',
+				  ],
+			  ];
 
 		yield 'with-other-attrs' => [
-			'test=value; Max-Age=3600; Path=/web',
-			['name' => 'test', 'value' => 'value', 'path' => '/web'],
-		];
+				  'test=value; Max-Age=3600; Path=/web',
+				  [
+					  'name'  => 'test',
+					  'value' => 'value',
+					  'path'  => '/web',
+				  ],
+			  ];
 
 		yield 'with-flags' => [
-			'test=value; Secure; HttpOnly; SameSite=Lax',
-			['name' => 'test', 'value' => 'value', 'secure' => true, 'httponly' => true, 'samesite' => 'Lax'],
-		];
+				  'test=value; Secure; HttpOnly; SameSite=Lax',
+				  [
+					  'name'     => 'test',
+					  'value'    => 'value',
+					  'secure'   => true,
+					  'httponly' => true,
+					  'samesite' => 'Lax',
+				  ],
+			  ];
```

Comment:
Revert the changes here. our codesniffer implementation cannot still handle gracefully these nested arrays but can become unsightly.

Reply:
Actually I didn't change this, maybe my IDE changed it, reverted

Reply:
Reverted more than once, but when I commit and pull, it auto reverse my changes

Reply:
Are you using VS Code?

Reply:
You can use the Git Staged Amend No Verify option. Then Push Force

Reply:
if not. directly in terminal
```
git commit --amend --no-verify
git push --force-with-lease
```

Reply:
Thank you